bindings/python/examples/how_tos/wallet/send_amount.py

```diff
@@ -4,7 +4,7 @@
 
 from iota_sdk import Wallet, WalletOptions, Utils, NodeIndexerAPI, SyncOptions, WalletSyncOptions, SendParams
 
-# In this example we send funds from an account wallet.
```

Comment:
This should stay, it's sending from an output owned by the account output

Reply:
The file should also be in the account_output folder

Reply:
Done